CMSIS/DSP/Testing/PatternGeneration/Support.py

```diff
@@ -152,7 +206,9 @@ def generatePatterns():
     writeTestsQ31(configq31)
     writeTestsQ15(configq15)
     writeTestsQ7(configq7)
-    
+
+    writeTests2(configf32,0)
```

Comment:
How is writeTestsF32 called ? It is not called in writeTests2. So patterns for F32 are no more regenerated

Reply:
As usual, it is cut in the preview, but writeTestsF32 is called. It's at line 205